source/module_elecstate/module_charge/charge.cpp

```diff
@@ -711,12 +711,45 @@ double Charge::check_ne(const double* rho_in) const
     Parallel_Reduce::reduce_pool(ne);
 #endif
     ne = ne * elecstate::get_ucell_omega() / (double)this->rhopw->nxyz;
-    std::cout << std::setprecision(10);
-    std::cout << " check the electrons number from rho, ne =" << ne << std::endl;
-    std::cout << std::setprecision(6);
+
     return ne;
 }
 
+void Charge::check_rho()
+{
```

Comment:
better to print out the "expected charge" and "charge afte renormalization", I suppose